KMDefaultSettings.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  KmDefaultSettingPreference.swift
+//  Kommunicate
+//
+//  Created by sathyan elangovan on 03/06/22.
+//
+
+import Foundation
+
+/// `KMDefaultSettings` is used for handling the default conersation assignee,bot,agent settings and storing the data
+public struct KMDefaultSettings {
```

Comment:
We already have a default config. Should we add another class for settings? Or maybe create another type for conversation settings and pass it in default config only.
https://docs.kommunicate.io/docs/ios-customization

Reply:
Moved all the settings to `defaultConfiguration`